eclair-core/src/test/scala/fr/acinq/eclair/channel/InteractiveTxBuilderSpec.scala

```diff
@@ -1652,21 +1657,31 @@ class InteractiveTxBuilderSpec extends TestKitBaseClass with AnyFunSuiteLike wit
       val initialBalanceA = commitmentA1.localCommit.spec.toLocal
       val initialBalanceB = commitmentA1.localCommit.spec.toRemote
       val amountPaid = 25_000_400 msat
-      val commitmentA2 = commitmentA1
+      val commitmentA1bis = commitmentA1
         .modify(_.localCommit.spec.toLocal).setTo(initialBalanceA - amountPaid)
         .modify(_.localCommit.spec.toRemote).setTo(initialBalanceB + amountPaid)
         .modify(_.remoteCommit.spec.toLocal).setTo(initialBalanceB + amountPaid)
         .modify(_.remoteCommit.spec.toRemote).setTo(initialBalanceA - amountPaid)
-      val commitmentB2 = commitmentB1
+      val commitmentA2bis = commitmentA2
+        .modify(_.localCommit.spec.toLocal).setTo(initialBalanceA - amountPaid)
+        .modify(_.localCommit.spec.toRemote).setTo(initialBalanceB + amountPaid)
+        .modify(_.remoteCommit.spec.toLocal).setTo(initialBalanceB + amountPaid)
+        .modify(_.remoteCommit.spec.toRemote).setTo(initialBalanceA - amountPaid)
```

Comment:
This is hiding a bug below where we compare balance with the replaced commitment instead of using the parent commitment. This is fixed in #2636

Reply:
Merged in [5fa5bf4](https://github.com/ACINQ/eclair/pull/2634/commits/5fa5bf4a5f1f159b3cba7083d4912b9103ad5740).